src/components/MostRecent.jsx

```diff
@@ -1,9 +1,14 @@
 import { Link } from "react-router-dom";
 
-function MostRecent({mostRecent}) {
+function MostRecent({mostRecent, query, topic}) {
     return (
       <article className="most-recent">
-        <h2>Recently Posted...</h2>
+        <h2> {query === "?sort_by=created_at&order=desc" ? `Recently posted ${topic ? `in ${topic}` : ""} ...` :
```

Comment:
Ooooh now I see why you did it how you did above, this is a lovely feature! (although you could maybe still have separate buttons and do something with && to check both.)